README.md

```diff
@@ -120,74 +120,37 @@ The easiest way is to create a proptery list in your application's bundle. By de
 
 ### Configuration object
 
-Alternatively, you can create a dictionary with the required values:
+Alternatively, you can create a configuration object ( `OktaAuthConfig`) from dictionary with the required values:
 
 ```swift
-let config = [
+OktaAuth.configuration = OktaAuthConfig(with: [
   "issuer": "https://{yourOktaDomain}/oauth2/default",
   "clientId": "{clientID}",
   "redirectUri": "{redirectUri}",
   "logoutRedirectUri": "{logoutRedirectUri}",
   "scopes": "openid profile offline_access",
   // Custom parameters
   "login_hint": "username@email.com"
-]
-```
+])
+``` 
 
 ## API Reference
 
 ### signInWithBrowser
 
-Start the authorization flow by simply calling `signIn`. By default, this method uses the values specified in the `Okta.plist` file:
-
-```swift
-OktaAuth
-  .signInWithBrowser()
-  .start(view: self)
-  .then { tokens in
-    // tokens.accessToken
-    // tokens.idToken
-    // tokens.refreshToken
-  }
-  .catch { error in
-    // Error
-  }
-```
-
-Alternatively, use a custom `plist` file using the `withPListConfig` argument:
-
-```swift
-OktaAuth
-  .signInWithBrowser()
-  .start(withPListConfig: "CustomPlist", view: self)
-  .then { tokens in
-    // tokens.accessToken
-    // tokens.idToken
-    // tokens.refreshToken
-  }
-  .catch { error in
-    // Error
-  }
-```
-
-Finally, use a dictionary instead of a `plist`:
+Start the authorization flow by simply calling `signIn`. 
 
 ```swift
-let config: [String: String] = [
-  // Your configuation
-]
-
-OktaAuth
-  .signInWithBrowser()
-  .start(withDictConfig: config, view: self)
-  .then { tokens in
+OktaAuth.signInWithBrowser(from: self) { tokens, error in
+    if let error = error {
```

Comment:
We're mixing `2` and `4` spaces as indentation across our examples, can you please standardize on one of them?

Reply:
Done.